src/components/organisms/UiMultipleAnswer/UiMultipleAnswer.vue

```diff
@@ -184,13 +183,13 @@ const hintType = computed<'error'|'default'>(() => (props.touched && props.inval
 watch(valid, (value) => {
   emit('update:invalid', !value);
 }, { immediate: true });
-const updateHandler = (newValue: string | unknown[] | Record<string, unknown>) => {
+const handleValueUpdate = (newValue: string | unknown[] | Record<string, unknown>) => {
```

Comment:
I thing about use this name convention https://jaketrent.com/post/naming-event-handlers-react/ WDYT?